scripts/test/shared.py

```diff
@@ -182,7 +182,7 @@ def is_exe(fpath):
             which('gcc') or which('clang'))
 NATIVEXX = (os.environ.get('CXX') or which('mingw32-g++') or
             which('g++') or which('clang++'))
-NODEJS = os.getenv('NODE', which('nodejs') or which('node'))
+NODEJS = os.getenv('NODE', which('node'))
```

Comment:
maybe `node or nodejs` (flip the order)? that would still handle old/odd distros where there is just `nodejs`.

Reply:
I thought about that.

Such users probably don't have a new  enough node anyway to run the binaryen tests.  

Remember this is only for running binaryen tests.  

Also, Debian and Ubuntu systems have /usr/bin/node being the nodejs binary anyway when nodejs is installed.

Also we have the "export NODE=xxx" method to override this is anyone really wants to.

Also, we have not such logic in emscripten .. we only look for `node` there, and emscripten is targeting a much wider group that the binaryen test suite.  If just looking for `node` is good enough there I think its good enough here.

Also, the original point of `nodejs` symlink I believe we because there was another package with a binary called node and this was a way to be sure you were getting nodejs over the other thing... and if we look for `node` first subvert the the entire point of that and risk getting the other thing called `node` which is nothing to do with JS. 

So, I think its cleaner all around.

Reply:
Ok, sgtm if you are sure this is just an issue in older distros. I worry maybe odd new ones do the same, but I don't have evidence of that.

Reply:
Actually I was wrong.. we do have logic in emscripten to look for nodejs..  I will hold off on this for a second.